server/data/reportingClient.ts

```diff
@@ -14,13 +14,13 @@ export default class ReportingClient {
     return new RestClient('Reporting API Client', config.apis.reporting, token)
   }
 
-  getCount(resourceName: string, token: string, filters: Dict<string>): Promise<number> {
+  getCount(resourceName: string, token: string, listRequest: ReportQuery): Promise<number> {
```

Comment:
Optional: Since the listRequest object is used in the getCount method only to get the recordWithFilterPrefix, then getCount could accept a recordWithFilterPrefix parameter of type Record<string,string> instead the entire ReportQuery object.

Reply:
That's true, but it's nice to keep the list and count endpoints consistent.